prediction_market_agent/db/sql_handler.py

```diff
@@ -0,0 +1,56 @@
+from loguru import logger
+from prediction_market_agent_tooling.tools.utils import check_not_none
+from sqlalchemy import Table, ColumnElement, Column, BinaryExpression
+from sqlmodel import create_engine, SQLModel, Session, select, desc, col, asc
+
+# from prediction_market_agent.db.models import LongTermMemories, Prompt
+import typing as t
+from prediction_market_agent.utils import DBKeys
+
+TypeTable = t.TypeVar("TypeTable", bound=SQLModel)
```

Comment:
This is needed otherwise using `Prompt` or `LongTermMemory` as return types of a downstream method doesn't match `SQLModel` and mypy complains.